Makefile

```diff
@@ -69,6 +77,33 @@ docker-build: check-docker docker-binary
 	docker build --rm -t ${IMAGE} rootfs
 	docker tag ${IMAGE} ${MUTABLE_IMAGE}
 
+# Building multi-arch docker images
+.PHONY: docker-build-all
+docker-build-all: BINDIR = ./rootfs
+docker-build-all: check-docker check-docker-experimental build-cross
+docker-build-all:
+	docker run --rm --privileged multiarch/qemu-user-static:register --reset
+	for arch in $(LINUX_ARCH); do \
+			cp $(BINDIR)/Dockerfile ./_dist/linux-$$arch -f; \
+			bin_arch=$$arch; \
+			if [ $$arch == arm64 ]; then \
+				bin_arch=aarch64; \
+			elif [ $$arch == 386 ]; then \
+				bin_arch=i386; \
+			fi; \
+			if [ $$arch != amd64 ]; then \
+				wget -O ./_dist/linux-$$arch/qemu-$$bin_arch-static \
+					https://github.com/multiarch/qemu-user-static/releases/download/v3.0.0/qemu-$$bin_arch-static; \
```

Comment:
Would you mind making the `wget` part of the RUN command instead of downloading qemu locally and COPYing it?